pytissueoptics/rayscattering/opencl/CLParameters.py

```diff
@@ -0,0 +1,73 @@
+from pytissueoptics.rayscattering.opencl.CLProgram import CLProgram
+import numpy as np
+
+
+class CLParameters:
+    dataPointSize = 16
+    photonSize = 64
+    seedSize = 4
+    materialSize = 32
+
+    def __init__(self, maxLoggerMemory: int = 1e8, workItemAmount: int = 100,
+                 photonAmount: int = 1000, loggerGlobalFactor: float = 0.75):
+        self._maxLoggerMemory = maxLoggerMemory
+        self._workItemAmount = workItemAmount
+        self._photonAmount = photonAmount
+        self._loggerGlobalFactor = loggerGlobalFactor
+
+    @property
+    def workItemAmount(self):
+        return np.int32(self._workItemAmount)
```

Comment:
These type conversions are very implicit and all over the place. I hope CLProgram can soon take this responsibility.